how-to-guides/multiple-integrations/notebooks/Multiple_integrations.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to use multiple Neptune integrations together?\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/neptune-ai/examples/blob/main/how-to-guides/multiple-integrations/notebooks/Multiple_integrations.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open in Colab\"/>\n",
+    "</a>\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://github.com/neptune-ai/examples/blob/main/how-to-guides/multiple-integrations\">\n",
+    "  <img alt=\"Open in GitHub\" src=\"https://img.shields.io/badge/Open_in_GitHub-blue?logo=github&labelColor=black\">\n",
+    "</a>\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://app.neptune.ai/o/showcase/org/Multiple-integrations/runs/details?viewId=standard-view&detailsTab=metadata&shortId=MUL-1\"> \n",
+    "  <img alt=\"Explore in Neptune\" src=\"https://neptune.ai/wp-content/uploads/2024/01/neptune-badge.svg\">\n",
+    "</a>\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://docs.neptune.ai/tutorials/multiple-integrations/\">\n",
+    "  <img alt=\"View tutorial in docs\" src=\"https://neptune.ai/wp-content/uploads/2024/01/docs-badge-2.svg\">\n",
+    "</a>\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "Neptune's integrations work independently of each other. This means that you can use multiple integrations together. 🦾\n",
+    "\n",
+    "For example, if you are doing hyperparameter optimization using Optuna on scikit-learn models, you can use both our Optuna and scikit-learn integrations together in the same script. The [Optuna integration](https://docs.neptune.ai/integrations/optuna/) will log all the relevant Optuna Study and Trial metadata, while the [scikit-learn integration](https://docs.neptune.ai/integrations/sklearn/) will log the scikit-learn model metadata. If you use Airflow to schedule the above setup, you can add our [Airflow integration](https://docs.neptune.ai/integrations/airflow/) to automatically log Airflow metadata 🪄\n",
+    "\n",
+    "However, since the integrations work independently, it is necessary to organize the logged metadata in a way that makes it easy for you to search for the information you need, without it being overwritten by another integration. ✅\n",
```

Comment:
```suggestion
    "However, since the integrations work independently, it is necessary to organize the logged metadata in a way that makes it easy for you to search for the information you need, without it being overwritten by another integration.\n",
```